packages/shared/src/components/cards/WelcomePostCardFooter.tsx

```diff
@@ -17,15 +17,31 @@ export const WelcomePostCardFooter = ({
     [post?.contentHtml],
   );
 
-  if (post.image) {
+  const image = useMemo(() => {
+    if (post?.image) return post?.image;
+
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(post?.contentHtml, 'text/html');
+    const imgTag = doc.querySelector('img');
+    if (imgTag) {
+      return imgTag.getAttribute('src') || null;
+    }
+
+    return null;
```

Comment:
No need to return `null`, `undefined` is just fine, also above with `|| null`.

Reply:
I have made this change